modules/libdeflate/1.18/patches/add_build_file.patch

```diff
@@ -0,0 +1,59 @@
+--- /dev/null
++++ b/modules/libdeflate/1.18/patches/BUILD.bazel
+@@ -0,0 +1,56 @@
++# SPDX-License-Identifier: BSD-3-Clause
++# Copyright (c) Contributors to the OpenEXR Project.
++
++licenses(["notice"])
++
++exports_files(
++    glob(["lib/**"]) + ["common_defs.h"],
++    visibility = ["//visibility:public"],
++)
++
++cc_library(
++    name = "libdeflate",
```

Comment:
The generated library will be called `liblibdeflate.so`. You could call the target `deflate ` and add a `libdeflate` alias for convenience.

Reply:
I changed the name to `deflate`